src/main/scala/com/twitter/diffy/proxy/RefineHttpHeadersByLabel.scala

```diff
@@ -0,0 +1,54 @@
+package com.twitter.diffy.proxy
+
+import com.twitter.finagle.Filter
+import com.twitter.finagle.http.Request
+import com.twitter.util.Future
+import org.jboss.netty.handler.codec.http.{HttpResponse, HttpHeaders, HttpRequest}
+import scala.collection.JavaConverters._
+
+object RefineHttpHeadersByLabel {
+  val knownLabels: Seq[String] = Seq("primary", "secondary", "candidate")
```

Comment:
These constants should be maintained in DifferenceProxy.scala to ensure that the labels always match. The current implementation requires these labels to be maintained in two places.